src/libs/nav.js

```diff
@@ -72,13 +75,14 @@ export const getLink = (...args) => `#${getPath(...args).slice(1)}` // slice off
  * @param args
  */
 export const goToPath = (...args) => {
-  history.push({ pathname: getPath(...args) })
+  const [pathname, search] = getPath(...args).split('?')
```

Comment:
TOL: as currently written, `getPath` won't return a query string, so this isn't doing anything. For completeness sake, we could give it a params argument, and have it do that. Alternatively, instead of having it construct a string and then parsing it, we could modify it to return a `location`-compatible object, which could be directly `push`-ed here.

Reply:
Good point. Reverting for now. Thoughts on the `getPath` signature to do this?

Reply:
The full signature for `goToPath` and `getLink` would be something like: `(key, pathParams, queryParams)`

Reply:
I'm inclined to file a bug to do this now rather than wait until we need it so we can get used to the alternative signature. Thoughts?